### ISSUE TITLE
Use `[com.fasterxml.jackson.core/jackson-core "2.0.0"]` to avoid conflicts with old versions

### ISSUE THREAD

**Issue by zoka** [dakrone/cheshire#22]
Hi,
I am developer of ringMon which is a Ring middleware that provides web browser REPL interface in various incarnations:
-  It can be inserted into web apps Ring chain
-  it can be added as a development dependency for non-web app
-  it is part of lein-webrepl,  a lein2  plugin that can start browser based REPL within any Clojure 1.3.0/lein 2.0 project in same manner that `lein repl` does, no changes to project.clj needed.

In all above use cases ringMon needs to fit in with existing projects, so to avoid conflicts dependencies were
kept to a minimum. It needs JSON encoder/decoder just to/from strings. Since I started with a Noir 1.2.2 application as a showcase for ringMon running on Heroku, I chose clj-json, already a Noir 1.2.2 dependency,  that was using `[org.codehaus.jackson/jackson-core-asl "1.5.0"]`.
All was fine until I needed to parse `project.clj`. Since clj-json can't do it, I switched to cheshire 3.0.0 that uses `[org.codehaus.jackson/jackson-core-asl "1.9.0"]`,  but then it does not work with Noir 1.2.2 since Noir indirectly depends on `[org.codehaus.jackson/jackson-core-asl "1.5.0"]`, that is unfortunately in conflict with 1.9.0 needed by cheshire :).

So,  I took a drastic action that did not feel right: cherry pick bits needed
from cheshire,  cobble them all in one file and declare dependency on jackson 1.5.0, so now embedded cheshire works fine with Noir 1.2.2. Today I tried to upgrade to Noir 1.3.0 beta and it turns out that Noir also uses cheshire
that is dependent on jackson 1.9.x and therefore incompatible with ringMon. 

Out of  desperation I decided to find the jackson source, change the package name and pull it in as well and then I had a moment of joy - somebody else has already done it:
[Jackson 2.0.0](https://github.com/FasterXML/jackson-core)

Basically, from 2.0.0 jackson has moved to GitHub and changed the package name to:
`[com.fasterxml.jackson.core/jackson-core "2.0.0"]`, it took me a very short time to get it working with embedded cheshire. Some configuration flags are  shuffled, but nothing major. [See it here](https://github.com/zoka/ringMon/blob/master/src/ringmon/json.clj)

Now ringMon is conflict free and it would be very nice if you could try this brand new version of jackson  in cheshire - so I can eventually stop embedding it into ringMon :)
Regards
Zoka
